### DISCUSSION TITLE
Welcome to SSVC Discussions!

### DISCUSSION THREAD

**Discussion by ahouseholder** [CERTCC/SSVC#87] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeroenh** [comment]
Thanks for the invitation. I'm Jeroen van der Ham, senior researcher at NCSC-NL and associate professor at UTwente.
I think this is an excellent and timely development, that is of interest to the wider CSIRT community. At NCSC-NL we're currently reconsidering how we deal with advisory data and this is a very important input to that process.

I've subscribed to the developments, but must admit that I have to catch up again and hope to provide some feedback on the document in the next few weeks.

**Reply by j---** [reply]
Thanks Jeroen, happy to have you helping and we'd welcome your feedback. Though the formatting isn't perfect, we're going to try to keep https://github.com/CERTCC/SSVC/blob/main/doc/ssvc_v2-0.html up to date as we merge new edits in to the various markdown files.
